dlpy/caffe_models/model_resnet101.py

```diff
@@ -87,13 +90,20 @@ def ResNet101_Model(s, model_table='RESNET101', n_channels=3, width=224, height=
                                     randomcrop=random_crop, offsets=offsets,
                                     randomFlip=random_flip, randomMutation=random_mutation))
 
+    input_data_layer = 'data'
```

Comment:
maybe add some point we need to convert these to dlpy models using functional api. of course this is for later